cdoc/cdoc_jar/src/main/java/org/cdoc4j/cli/Tool.java

```diff
@@ -0,0 +1,164 @@
+package org.cdoc4j.cli;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.security.GeneralSecurityException;
+import java.security.cert.X509Certificate;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+
+import org.cdoc4j.CDOC;
+import org.cdoc4j.CDOCBuilder;
+import org.esteid.sk.IDCode;
+import org.esteid.sk.LDAP;
+
+import static spark.Spark.get;
+import static spark.Spark.port;
+import static spark.Spark.post;
+
+
+
+
+public class Tool {
+
+
+    public static void main(String[] args) {
+        port(4444);
```

Comment:
Interesting port choice, why not 80 and/or why not take it from env var in compose?

Reply:
Duplicate